statsd/statsd.go

```diff
@@ -133,14 +137,15 @@ func AddFlags(fs *pflag.FlagSet) {
 	fs.Int(ParamMaxReaders, DefaultMaxReaders, "Maximum number of socket readers")
 	fs.Int(ParamMaxWorkers, DefaultMaxWorkers, "Maximum number of workers to process metrics")
 	fs.Int(ParamMaxQueueSize, DefaultMaxQueueSize, "Maximum number of buffered metrics per worker")
+	fs.Int(ParamMaxConcurrentEvents, DefaultMaxConcurrentEvents, "Maximum number of events sent concurrently")
 	fs.String(ParamMetricsAddr, DefaultMetricsAddr, "Address on which to listen for metrics")
 	fs.String(ParamNamespace, "", "Namespace all metrics")
 	fs.String(ParamWebAddr, DefaultWebConsoleAddr, "If set, use as the address of the web-based console")
 	//TODO Remove workaround when https://github.com/spf13/viper/issues/112 is fixed
 	// https://github.com/spf13/viper/issues/200
 	fs.String(ParamBackends, strings.Join(DefaultBackends, ","), "Comma-separated list of backends")
-	fs.Uint(ParamMaxCloudRequests, DefaultMaxCloudRequests, "Maximum number of cloud provider requests per second")
-	fs.Uint(ParamBurstCloudRequests, DefaultBurstCloudRequests, "Burst number of cloud provider requests per second")
```

Comment:
Unsigned flags are not supported by Viper (there is a bug I reported a while ago). Not sure how this worked.